docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -446,6 +446,7 @@ Below is the list of the supported config properties.
 | `theme`                       | `system`                                                                                                             | Color theme for the app. Available modes: `light`, `dark`, `system`.                                       |
 | `terminal.fontFamily`         | `Menlo, Monaco, monospace` on macOS<br/>`Consolas, monospace` on Windows<br/>`'Droid Sans Mono', monospace` on Linux | Font family for the terminal.                                                                              |
 | `terminal.fontSize`           | 15                                                                                                                   | Font size for the terminal.                                                                                |
+| `terminal.windowsUseConpty`   | true                                                                                                                 | Whether to use the ConPTY system instead of winpty. ConPTY requires Windows version 18309 and above.       |
```

Comment:
```suggestion
| `terminal.windowsUseConpty`   | true                                                                                                                 | Whether to use the ConPTY system instead of winpty. ConPTY requires Windows 10 19H1 or above.              |
```

At first I wanted to point out that we should probably say "Windows 10" here, but then I dug a little bit deeper.

18309 is the name of a preview build of Windows 10. According to [the 18309 release post](https://blogs.windows.com/windows-insider/2019/01/03/announcing-windows-10-insider-preview-build-18309/) and [the Wikipedia article](https://en.wikipedia.org/wiki/Windows_10_version_history#Version_1903_(May_2019_Update)), Windows 10 19H1 would be the earliest stable version of Windows 10 which supports ConPTY.

Reply:
Another option is not to have a flag for conpty but rather for winpty.

ConPTY is the default which we want people to use. The flag, I think, should be for the older backend that should be used instead of ConPTY in certain circumstances.

Another argument for that would be that let's say another backend is added. If `useConpty` is set to `false`, which backend should we use? This is less about this specific situation with the PTY backend and more about picking names for the config options which are future-proof.

So what about something like this:

```suggestion
| `terminal.windowsBackend` | `auto` | `auto` uses modern [ConPTY](https://devblogs.microsoft.com/commandline/windows-command-line-introducing-the-windows-pseudo-console-conpty/) if available, which requires Windows 10 (19H1) or above. Set to `winpty` to use winpty even if ConPTY is available. |
```

(I don't think we have to care about the `|` markers in the table being properly aligned)

Reply:
`auto` | `winpty` is much better, thank you!